scripts/build/fix.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
```

Comment:
@markus-walther why do you write a script for something that git revert does for you? In case of a broken build, better revert via git instead of writing a script that basically does the same

Reply:
Reason: keep repo as clean as possible.

Reply:
Because I don't want to do precision git reverts on package.jsons , taking care that other stuff is *not* reverted. And because my script fixes the underlying problem, so can be trusted - unlike git revert, which knows *nothing* about the problem, so might produce inadvertant other results.

Reply:
@markus-walther i have no strong opinion on this one, seems just not necessary but if you prefer it so, then ok to me

Reply:
And also, that the build breaks is a very rare case so that precision work would be very rare

Reply:
tests will fail anyway if you try to build the project with the `main` field there